apps/etterlatte-migrering/src/main/kotlin/migrering/verifisering/Verifiserer.kt

```diff
@@ -77,8 +77,8 @@ internal class Verifiserer(
                     val foedselsdato: LocalDate =
                         person.getOrNull()?.foedselsdato?.verdi
                             ?: request.soeker.getBirthDate()
-                    if (foedselsdato.isBefore(LocalDate.of(2006, Month.JANUARY, 1))) {
-                        logger.warn("Søker er over 18 år")
+                    if (foedselsdato.isBefore(LocalDate.of(2006, Month.JANUARY, 1)) && !request.dodAvYrkesskade) {
+                        logger.warn("Søker er over 18 år og det er ikke yrkesskade")
```

Comment:
Trengs det en egen sjekk på alder for yrkesskade også, eller har vi kontroll på det (20 år)?

Reply:
> Trengs det en egen sjekk på alder for yrkesskade også, eller har vi kontroll på det (21 år)?

Kikka i uttrekket no, dei to eldste har fødselsmånad 2003.01 og 2003.03

Den eldste er jo litt interessant da, usikker på om vi skal migrere den?

Reply:
Litt usikker på hvor cut-offen skal gå jeg også 🤔

Reply:
Tar det på Slack, og så legg eg dei andre klar til køyring så lengje.